docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -99,12 +102,16 @@ public class WriteHealthBehaviour : MonoBehaviour
 
 1. Declare a field of type `Health` Writer and decorate it with a `[Require]` attribute. 
 
-2. Register a callback for `Reader.ComponentUpdated(ISpatialComponentUpdate update) +=` or for `Reader.<component property name>Updated() +=` during `OnEnable()`.
+2. Register a callback for `Reader.ComponentUpdated +=` or for `Reader.<component property name>Updated +=` during `OnEnable()`.
     *  `Reader.ComponentUpdated` is invoked when any component property is updated.
     *  `Reader.<component property name>Updated` is invoked when a specific component property is updated.
 
 **Note:** 
-`Reader.ComponentUpdated` callbacks are invoked before specific property update callbacks. Callbacks can be deregistered using `Reader.ComponentUpdated(ISpatialComponentUpdate update) -=` and `Reader.<component property name>Updated() -=`. Callbacks are also automatically deregistered when a Reader or Writer is removed. Do not deregister callbacks during `OnDisable()` as that’s an invalid operation.
+`Reader.ComponentUpdated` callbacks are invoked before specific property update callbacks. You can deregister callbacks using `Reader.ComponentUpdated -=` and `Reader.<component property name>Updated -=`. The SpatialOS GDK also automatically deregisters all callbacks of a Reader or Writer upon validating them when requirements are not met anymore. Do not deregister callbacks during `OnDisable()` as that’s an invalid operation.
+
+**Known issue warning:**
+- The `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property has changed even when it has not. This is unintended behavior which we will fix in an upcoming update.
+- `Reader.<component property name>Updated` may be invoked even if `<component property name>` did not truly change. This is unintended behavior which we will fix in an upcoming update.
```

Comment:
"truly" reads a bit oddly; we would tend to use "really". I think we can say:
"...did not change."

"Truly" is quite a slightly emotive term.

Reply:
Removed "truly"